spec/Section 4 -- Introspection.md

```diff
@@ -348,6 +387,8 @@ Fields
 
 * `kind` must return `__TypeKind.LIST`.
 * `ofType`: Any type.
+* `punctuatedName` must return `'[' + ofType.punctuatedName + ']'`.
```

Comment:
@IvanGoncharov Using `+` for string concatenation in the spec is likely not allowed since it's a cross-language specification. Do we have any examples in the spec where concatenation is used that we could apply here?

Reply:
Same thing goes for `ofType.punctuatedName` for referencing a field of a value. I think we might have to settle for writing this in prose, awkward though this may be:

```suggestion
* `punctuatedName` must return the result of concatenating the `[` character, the wrapped type's `punctuatedName` value, and a `]` character.
```